> Did the first version (the one without the lowerCamelCase naming) compile? 

I've just checked it out — nope, it doesn't compile. Neither does the current one, because of the following error:
```
src/provider.ts:43:35 - error TS2304: Cannot find name 'TextDocument'.

43 export function getAsmUri(source: TextDocument): Uri {
                                     ~~~~~~~~~~~~
```

It's wise to test the code before pushing it. :slightly_smiling_face: 

To prevent such things from happening in the future, I've set up a Github Actions. Now it should be a little bit more transparent whether PR is buildable or not.

--- a/src/provider.ts
+++ b/src/provider.ts
@@ -1,6 +1,6 @@
 'use strict';
 
-import { workspace, languages, Uri, EventEmitter, TextDocumentContentProvider, Event } from 'vscode';
+import { workspace, languages, Uri, EventEmitter, TextDocumentContentProvider, Event, TextDocument } from 'vscode';
 import * as Path from 'path';
 import { AsmDocument } from './document';
 

